content/desktop/hardened-desktop/settings-management/air-gapped-containers.md

```diff
@@ -0,0 +1,86 @@
+---
+description: Learn how to create air-gapped containers with Settings Management
+title: Configure air-gapped containers with Settings Management
+keywords: settings management, air gapped, security, Docker Desktop, configuration, proxy, network
+---
+
+> **Beta feature**
+>
+> This feature is in [Beta](../../../release-lifecycle.md/#beta).
+> It's available with Docker Desktop version 4.29 and later.
+{ .experimental }
+
+Air-gapped containers allows administrators to restrict containers from accessing network resources, limiting where data can be uploaded to or downloaded from.
+
+Docker Desktop can apply a custom set of proxy rules to network traffic from containers. The proxy can be configured to:
+
+- Allow network connections
+- Reject network connections
+- Tunnel through an HTTP or SOCKS proxy
+
+You can choose:
+
+- Which outgoing TCP ports the policy applies to. For example, only certain ports, `80`, `443` or all with `*`.
+- Whether to forward to a single HTTP or SOCKS proxy, or to have a policy per destination via a Proxy Auto-Configuration (PAC) file.
+
+## Configuration
+
+Assuming enforced sign-in and Settings Management are enabled, add the new proxy configuration to the `admin-settings.json` file. For example:
+
+```json
+{
+  "configurationFileVersion": 2,
+  "containersProxy": {
+    "locked": true,
+    "mode": "manual",
+    "pac": "http://192.168.1.16:62039/proxy.pac",
+    "transparentPorts": "*"
+  }
+}
+```
+
+The `containersProxy` setting describes the policy which is applied to traffic from containers. The valid fields are:
+
+- `locked`: If true, it is not possible for developers to override these settings. If false the settings are interpreted as default values which the developer can change.
+- `mode`: Same meaning as with the existing `proxy` setting. Possible values are `system` and `manual`.
+- `http`, `https`, `exclude`: Same meaning as with the `proxy` setting. Only takes effect if `mode` is set to `manual`.
```

Comment:
These fields are not shown in the example above. I think it would be clearer if they were.

Reply:
If added to the example above they would be
```
    "http": "",
    "https": "",
    "exclude": "",
```